closure/testing/test/BUILD

```diff
@@ -31,28 +31,31 @@ closure_js_library(
     deps = ["//closure/library"],
 )
 
-closure_js_test(
-    name = "arithmetic_module_test",
-    srcs = ["arithmetic_module_test.js"],
-    defs = ["--entry_point=goog:arithmetic_module_test"],
-    language = "ECMASCRIPT6_STRICT",
-    deps = [
-        ":arithmetic_module_lib",
-        "//closure/library",
-        "//closure/library:testing",
-    ],
-)
-
-closure_js_test(
-    name = "arithmetic_scope_test",
-    srcs = ["arithmetic_scope_test.js"],
-    language = "ECMASCRIPT6_STRICT",
-    deps = [
-        ":arithmetic_module_lib",
-        "//closure/library",
-        "//closure/library:testing",
-    ],
-)
+# TODO(hochhaus): Enable test when upstream local file loader fix is included
```

Comment:
It isn't safe in the event they load files off of disk. For production deployments it shouldn't be an issue. It is a great point that this is a serious problem and it would be good to isolate our users.

Is a new release planned soon? If so, waiting is a great option. If not, we could also apply the loader bugfix from a patch file.